docs/manifest/equivalentaddins.md

```diff
@@ -36,6 +39,41 @@ For more information, see [Version overrides in the add-in only manifest](/offic
 
 - [EquivalentAddin](equivalentaddin.md)
 
+## Can contain
+
+The **\<EquivalentAddins\>** element can contain the following child element.
+
+|Element|Content|Mail|TaskPane|
+|:-----|:-----:|:-----:|:-----:|
+|[Effect](#effect)|No|No|Yes|
+
+### Effect
+
+Specifies either that the COM add-in is disabled and hidden (instead of the Office Add-in) when they conflict, or specifies that the user chooses which to disable and hide. There are two possible values.
+
+- **DisableWithNotification**: All of the COM add-ins specified in the child **\<EquivalentAddin\>** elements will be disabled and hidden. A popup dialog notifies the user that this happening.
+- **UserOptionToDisable**: The user is prompted to choose whether to disable and hide COM add-ins specified in the child **\<EquivalentAddin\>** elements or to disable and hide the Office Add-in.
```

Comment:
I think the element name can be place in backticks instead of bold - here and other instances

Reply:
> I think the element name can be place in backticks instead of bold - here and other instances

This is the pattern we settled on a while back.

Reply:
True. We'll need to revise it since the updated guidance is to use backticks

Reply:
> True. We'll need to revise it since the updated guidance is to use backticks

OK. That's out-of-scope for this PR. I'll create a backlog item.